companies/views.py

```diff
@@ -213,9 +213,12 @@ def statistics(request, year):
 
 @permission_required('companies.base')
 def companies_list(request, year):
-	fair = get_object_or_404(Fair, year = year)
-
 	form = CompanySearchForm(request.POST or None)
+	if form.is_valid():
```

Comment:
I think we should make sure the current fair is still the current year's fair. So that contract and responsibilities are still for the current one (now 2019).

Instead we need to filter the company list based on what companies were exhibitors for another year. But still show the info for the current year.